FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/FhirToDataLakeOrchestratorJob.cs

```diff
@@ -382,83 +387,110 @@ private async Task<List<PatientWrapper>> GetToBeProcessedPatientsAsync(Cancellat
             return nexTimeOffset;
         }
 
-        private async Task CheckRunningJobComplete(IProgress<string> progress, CancellationToken cancellationToken)
+        private async Task CheckFirstRunningJobComplete(IProgress<string> progress, CancellationToken cancellationToken)
```

Comment:
Just curious, should we update this logic when update splitting job by resource type?
Since we want to keep consistent among different resource types and commit all jobs one time.

Reply:
Maybe we could have further discussion about how to commit jobs when update splitting jobs by resource type.